twisted/conch/scripts/ckeygen.py

```diff
@@ -33,7 +33,8 @@ class GeneralOptions(usage.Options):
                      ['type', 't', None, 'Specify type of key to create.'],
                      ['comment', 'C', None, 'Provide new comment.'],
                      ['newpass', 'N', None, 'Provide new passphrase.'],
-                     ['pass', 'P', None, 'Provide old passphrase.']]
+                     ['pass', 'P', None, 'Provide old passphrase.'],
+                     ['format', 'o', 'sha256-base64', 'Fingerprint format of key file.']]
 
```

Comment:
The latest version is just adding stuff.

so now we have

```
COMMAND --fingerprint --format=SOMETHING
```

I think that in the previous proposed version it was suggested to convert the `fingerprint` flag to an parameter

```
COMMAND --fingerprint=FORMAT
```

in trunk we now have fingerprint as a flag

```
COMMAND --fingerprint
```